Makefile

```diff
@@ -894,7 +898,7 @@ endif
 CMAKE_DIR_GCC := $(HOME)/pyston-build-gcc
 CMAKE_SETUP_GCC := $(CMAKE_DIR_GCC)/build.ninja
 $(CMAKE_SETUP_GCC):
-	@$(MAKE) cmake_check
+	@$(MAKE) clang_check
```

Comment:
You are right. Sorry for the silly error.

Not quite familiar with github workflow.
Should I add a new commit after your view or change the commit to the correct one? (I choose the latter, but it's troublesome than the former)